openerp/openupgrade/openupgrade.py

```diff
@@ -346,12 +346,33 @@ def write_value(ids, field, value):
                 # Iterating over ids here as a workaround for lp:1131653
                 obj.write(cr, SUPERUSER_ID, [res_id], {field: value})
         else:
-            cr.execute(
-                """
-                UPDATE %s
-                SET %s = %%s
-                WHERE id IN %%s
-                """ % (obj._table, field), (value, tuple(ids)))
+            query, params = "UPDATE %s SET %s = %%s WHERE id IN %%s" % (
+                obj._table, field), (value, tuple(ids))
+            # handle fields inherited from somewhere else
+            if field not in obj._columns:
+                query, params = None, None
+                for table in obj._inherits:
+                    if obj._inherit_fields[field][0] != table:
+                        continue
+                    col = obj._inherits[table]
+                    # this is blatantly stolen and adapted from
+                    # https://github.com/OCA/OCB/blob/def7db0b93e45eda7b51b3b61
```

Comment:
OK, you're right. Then we can put the line all together and put the noqa flag to avoid the check in that concrete line. Anyways, please check first my other comment changing the approach.